bin/run-all

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
`#!/usr/bin/env sh` will be more flexible on different systems. See scripts in the Linkerd2 repo's `bin` folder.